gengokit/template/NAME-service/cmd/NAME-server/main.gotemplate

```diff
@@ -21,9 +21,9 @@ import (
 
 	// This Service
 	handler "{{.ImportPath -}} /handlers"
```

Comment:
`handler` should be removed as well, and all places that currently call `handler` should be updated to `handlers`. Could you roll that into this pull?